CONTRIBUTING.md

```diff
@@ -71,6 +72,52 @@ When afraid to touch force and risk losing your work (do backups!..),
 try *merging master into your branch* instead of rebasing onto it.
 This is discouraged, as it produces ugly hard-to-maintain commit history.
 
+## Commit message rules
+
+When commiting your changes into the repository you may want to follow some
+rules to make the git history more readable and consistent:
+
+* Start the message capitalized (only the first character must be in uppercase)
+* Be short and concise, the whole concept must fit one line
+* If a command is inlined, use backticks
+* Add a double-hashtag if the change matters for the changelog (See below)
+* If the commit fixes a bug start with 'Fix #number - '
```

Comment:
Actually... What about putting the `Fix #number` in the third line? I think it becomes hard to put a concept in one single line if you have `Fix #XXXXX - something (#123131312) ##sometag`. There are already way too many things IMHO. WDYT?

Reply:
Agree, it makes a better sense. We could even use the same approach as with "Signed-off: bla-bla" - just put it on the last line.

Reply:
Multiline commits are a pain to read and parse. If the info is not relevant or its just detailed info about the first line lgtm. But the first line should describe everything contained in the commit without going too deep into details.

Reply:
I think the `Fix #number` should be in the first line. We want to encourage people to fix issues (this is monetarily-unpaid technical support after all) and having the fixes prominently displayed in the commit list is, imho, one small way of saying thank you.

Besides, it's not really a problem to have an overly long first line that just spills over into a second line.

Reply:
Actually it is a problem, because the first line can only be 72 (or  80, I don't remember) chars long and it is the "title" of the commit/PR (what you see when you use `git log --oneline`, for example). I'm ok with keeping it in first line as well though.

Reply:
i dnt think gh obeys the Fix # in any other line, but never tried